packages/framework/src/ShopsysFrameworkBundle.php

```diff
@@ -16,6 +17,7 @@ public function build(ContainerBuilder $container)
         parent::build($container);
 
         $container->addCompilerPass(new RegisterCronModulesCompilerPass());
+        $container->addCompilerPass(new FriendlyUrlDataProviderCompilerPass());
```

Comment:
now when I see the other compiler passes, should not be the name of the class be rather `RegisterUrlDataProvidersCompilerPass`?